src/plays/color-palettes-generator/ColorPalettesGenerator.js

```diff
@@ -0,0 +1,68 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import { useState, useEffect } from 'react';
+import SingleColor from './components/SingleColor';
+import { getcolorsarray, alertToast } from './components/utils';
+
+function ColorPalettesGenerator(props) {
+  const [color, setColor] = useState('#e01017');
+  const [error, setError] = useState(false);
+  const [colorlist, setColorList] = useState();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    try {
+      if (color.length == 7) {
+        // getcolorsarray function imported from Utils.js
+        let colors = getcolorsarray(color);
+        setColorList(colors);
+        setError(false);
+      } else {
+        setError(true);
+        // Imported from utils.js
+        alertToast('Please enter a valid hex code');

```

Comment:
This will directly go into catch block and there we can handle the error.
```suggestion
       throw 'Please enter a valid hex code'
```

Also store this message in constant 

```
Please enter a valid hex code
```

Reply:
Okay Done